java-sdk/connectors/random-connector-incremental/build.gradle

```diff
@@ -0,0 +1,4 @@
+dependencies {
+  // Note: Fusion libraries are added by parent project.
+  compile project(":random-connector")
```

Comment:
Instead of building a separate connector - could we add it into the same example? or what problems do you see adding this part to the same project?